src/uct/ib/dc/dc_mlx5.h

```diff
@@ -196,8 +198,8 @@ struct uct_dc_mlx5_iface {
         int16_t                   available_quota;             /* if available tx is lower, let
                                                                   another endpoint use the dci */
         /* LIFO is only relevant for dcs allocation policy */
-        uct_dc_mlx5_dci_pool_t    dci_pool[UCT_DC_MLX5_IFACE_MAX_LAG_PORTS];
-        uint8_t                   num_lag_ports;
+        uct_dc_mlx5_dci_pool_t    dci_pool[UCT_DC_MLX5_IFACE_MAX_DCI_POOLS];
+        uint8_t                   num_dci_pools;
```

Comment:
can we move it near line 196 to close a hold?

Reply:
all uint8/16 vars should be together to close the hold
also we can make policy a u8, gcc does not compress enums by default